### DISCUSSION TITLE
Fullscreen button doesn't work on mobile

### DISCUSSION THREAD

**Discussion by zahry7** [mistic100/Photo-Sphere-Viewer#1296] [General]
Hey, I have an issue with the fullscreen button. It works perfectly on a desktop, even in a minimized window. But it's not responding to a 'tap' on mobile devices. I have tried different browsers (Chrome, Safari). Not even the demo on the official PSV website is not working on mobile. The description button is working fine.
Has anyone experienced a similar issue? Thanks in advance.

**Comment by mistic100** [comment]
If you are on iOS, blame Apple  
https://github.com/mistic100/Photo-Sphere-Viewer/issues/985

If not, please open a new issue (not a discussion)
